### DISCUSSION TITLE
XML during upgrade

### DISCUSSION THREAD

**Discussion by musicpanda** [PrestaShop/PrestaShop#37663] [Q&A] [answered]
I want to upgrade a shop to version 8.1.7 - so not the latest version.

I put the zip file in the admin/autoupgrade/download folder. Unfortunately the software also asks for an XML file but it gives no indication where I can find this file and what its function is. Yet without this file it is impossible to proceed. The documentation on prestashop-project says nothing on the subject. Does anyone know more?

**Comment by Hlavtox** [comment, marked as answer]
@musicpanda The XML file it wants is the one attached to the release - https://github.com/PrestaShop/PrestaShop/releases/tag/8.1.7, it contains checksums of the zip file, the autoupgrade checks it to verify the integrity of the zip.

![Snímek obrazovky 2024-12-14 172623](https://github.com/user-attachments/assets/48781dc7-7a34-483e-a733-f2f4c5c9c1ff)

**Comment by musicpanda** [comment]
Thank you

**Comment by 01Kuzma** [comment]
The 9.0.x does come with XML? Now way to use the local channel anymore?
